### PR TITLE
Reorganize actions

### PR DESCRIPTION
**Pull request by jimscarver**
This is a draft until it supports testnet.

This branch if for bootstrapping, chrckpointing and updating a liquid democracy RChain RNode localhost instance for linux and Window10 WSL2..

### CODE REVIEW

File: inbox.rho [hunk 1]
```diff
@@ -8,7 +8,7 @@ contract Inbox(capabilities) = {
     box!(Nil) |
 
     contract read(ret) = {
-      for (items <<- box) {
+      for (items <- box) {
```
Comment (dckc) [inline inbox.rho:11]: yikes! That's a critical bug! Maybe put it in a PR of its own?

Comment (dckc) [reply inbox.rho:11]: oh... I read it the other way... never mind. it's fine here.

**Comment by jimscarver** [comment]
What this pull will dois create a complete usable development system, once testnet works again. While there are pieces elsewhere and this may not be the final resting place, this is not a docker solution and is specific to the capability directory structure of communities/working groups and their communications and their governance.

I'll try to update contributing.md accordingly.

**Comment by jimscarver** [comment]
> how does the UI code handle `set` in `FieldSpec`? I wonder if it still typechecks.

it still works the same. We separated rho files, not typechecks yet.

**Review by dckc — CHANGES_REQUESTED** [review]
Would you please check that `yarn lint:types` and `yarn lint-check` both pass?

See also https://github.com/rchain-community/liquid-democracy/issues/12#issuecomment-807286310

**Comment by dckc** [comment]
Is this really all intended to go in one PR? 75 commits, 60 files, 2000LOC? I can't imagine effectively reviewing such a thing.

**Comment by jimscarver** [comment]
This is a reorganization with new build and testing that must all work together or not. When we get through a complete test scenario successfully we will have reviewed it all.

**Comment by jimscarver** [comment]
Voting is working again finally so a merge is eminent once the selection and uri mapping for localhost, testnet and mainnet is working of close enough to be  used. Bill, Tom and I are far more comfortable with shell than node.js so maintaining the scripts should not be an issue. 

@Bill-Keuntje will be scarce for a couple weeks and @w2vy is still coming up to speed. I'll try to  get testnet working well enough so we can merge before the network selection is fully implemented. Tom is kelping to test.

**Comment by jimscarver** [comment]
changing to the result channels in Issue.rho tally got "Deploy error when executing Rholang code." now with the old Issue.rho tally is also failing. :frowning:

Is it just me?

**Comment by dckc** [comment]
@jimscarver writes in https://github.com/rchain-community/liquid-democracy/issues/12#issuecomment-817368517
> neither lint-check or yarn lint:types pass in master.

OK; I withdraw this request for changes.

**Review by Bill-Keuntje — APPROVED** [review]
Tested and working

**Pull request merged**